event_source_mappings.tf

```diff
@@ -44,6 +44,7 @@ resource "aws_lambda_event_source_mapping" "event_source" {
   parallelization_factor             = lookup(each.value, "parallelization_factor", null)
   starting_position                  = lookup(each.value, "starting_position", length(regexall(".*:sqs:.*", lookup(each.value, "event_source_arn", null))) > 0 ? null : "TRIM_HORIZON")
   starting_position_timestamp        = lookup(each.value, "starting_position_timestamp", null)
+  function_response_types            = lookup(each.value, "function_response_types", [])
```

Comment:
I tested this out defaulting to both an empty list `[]` and to `null`. Both behaved the same. Do you have a preference?

Reply:
@Schwamkrug `null` would be a bit more consistent to the other configuration values, so I would lean to that.

Reply:
Sounds good. Fixed!